independent-publisher-connectors/Airly/apiDefinition.swagger.json

```diff
@@ -0,0 +1,1942 @@
+                            },
+                            "percent": {
+                              "type": "number",
+                              "format": "float",
+                              "description": "Concentration value of a given pollutant expressed as a percentage of this concentration in the standard / limit, e.g. for a standard of 50\u03bcg/m3 of PM10 the concentration of 25\u03bcg/m3 is expressed as 50%.",
+                              "title": "Percent - history"
+                            },
+                            "averaging": {
+                              "type": "string",
+                              "description": "Measurement values are interpolated by averaging measurements from nearby sensors (up to 1,5km away from the given point).",
+                              "title": "Averaging - history"
+                            }
+                          }
+                        }
+                      },
+                      "standards": {
+                        "x-ms-visibility": "advanced",
+                        "description": "Standards - history.",
+                        "type": "array",
+                        "items": {
+                          "x-ms-visibility": "advanced",
+                          "description": "List of standards - history.",
+                          "type": "object",
+                          "properties": {
+                            "name": {
+                              "type": "string",
+                              "description": "Name of this standard. Currently only WHO standards are supported.",
+                              "title": "Standard name - history"
+                            },
+                            "pollutant": {
+                              "type": "string",
+                              "description": "Name of the pollutant to which this standard applies, currently only PM10 and PM25 are supported.",
+                              "title": "Pollutant - history"
+                            },
+                            "limit": {
+                              "type": "number",
+                              "format": "float",
+                              "description": "Certain average concentration level of the pollutant which should not be exceeded over given period of time; in other words, this is the value for which the pollutant concentration reaches 100% of the limit.",
+                              "title": "Limit - history"
+                            },
+                            "percent": {
+                              "type": "number",
+                              "format": "float",
+                              "description": "Concentration value of a given pollutant expressed as a percentage of this concentration in the standard / limit, e.g. for a standard of 50\u03bcg/m3 of PM10 the concentration of 25\u03bcg/m3 is expressed as 50%.",
```

Comment:
@tposzytek Unicodes - The connector's swagger still have unicodes which may not be supported in all the Power Automate page, Can you please replace all the unicodes with respective characters in the English language.